webrender_traits/src/api.rs

```diff
@@ -207,8 +207,8 @@ impl RenderApi {
     }
 
     /// Translates a point from viewport coordinates to layer space
-    pub fn translate_point_to_layer_space(&self, point: &Point2D<f32>)
-                                          -> (Point2D<f32>, PipelineId) {
+    pub fn translate_point_to_layer_space(&self, point: &LayoutPoint)
```

Comment:
hmm, this one makes me worried. The function is supposed to translate a point from one space to another, yet our type system does not reflect that here, using `LayoutPoint` on both ends